cli/args.go

```diff
@@ -29,6 +30,7 @@ func InitArgs() (Args, bool) {
 	fs.StringVar(&args.Filter, "filter", ".*", "Only apply tags to the selected resource types (regex)")
 	fs.BoolVar(&args.Verbose, "verbose", false, "Enable verbose logging")
 	fs.BoolVar(&args.Rename, "rename", true, "Keep the original filename or replace it with <basename>.terratag.tf")
+	fs.BoolVar(&args.Terragrunt, "terragrunt", false, "Tags all the terraform files under .terragrunt-cache")
```

Comment:
Why do we need this to be optional?
I mean, couldn't we simply identify if there's a cache folder - and if there is, tag its content?

One reason that I thought of - is to alert the user that he needs to run `init` before using `terratag` in the use case of `terragrunt`.
If that was your thought - I'd suggest to introduce a more generic `type` flag that defaults to `terraform` so we can later even support other IaC types

Reply:
Yes, that was my thought.
I will modify it to "type".